app/demangler.cpp

```diff
@@ -38,6 +39,7 @@ Demangler::Demangler()
 {
     loadDemangleLib(QStringLiteral("rustc_demangle"), "rustc_demangle", QByteArrayLiteral("_R"));
     loadDemangleLib(QStringLiteral("d_demangle"), "demangle_symbol", QByteArrayLiteral("_D"));
+    m_demanglers.push_back({"$s", reinterpret_cast<Demangler::demangler_t>(swift_demangle)});
```

Comment:
why is this not using `loadDemangleLib` like the above?

Reply:
swift_demangle can be compiled as a standalone shared library and used the same way as you use rustc_demangle or d_mangle.

Reply:
this does not answer my question. can we or can we not reuse `loadDemangleLib`? if not, why not?